src/main/java/io/openepcis/convert/Conversion.java

```diff
@@ -0,0 +1,90 @@
+package io.openepcis.convert;
+
+import io.openepcis.constants.EPCISFormat;
+import io.openepcis.constants.EPCISVersion;
+import lombok.Getter;
+
+@Getter
+public class Conversion {
+
+    private EPCISFormat fromMediaType;
+    private EPCISVersion fromVersion;
+    private EPCISFormat toMediaType;
+    private EPCISVersion toVersion;
+    private boolean generateGS1CompliantDocument;
+
```

Comment:
please re-introduct the public static Conversion of(....) methods. The make it easier to refactor existing code, and still make sense to directly create the required kind of instance.